docs/assignment2.md

```diff
@@ -1 +1,409 @@
+| F10  | Rooms      | There is a specific number of rooms that the player can enter, each resembling a room at the VU. |
+
+### 
+
+### Used modeling tool
+
+We are using draw.io
+
+###
+
+## Class diagram
+
+Author(s): Eduardo Lira & Theresa Schantz
+
+This chapter contains the specification of the UML class diagram of your system, together with a textual description of all its elements.
+
+```
+Class Diagram of VuORK
+```
+
+> Figure representing the Class Diagram of VuORK (Diagram 1) **DESCRIPTIVE**
+
+![Class Diagram of VuORK](https://i.postimg.cc/DZX98JfB/Class-diagram-Class-diagram.png)
+
+<h5 id="Item">Item</h5>
+
+The Item class is an abstract type which can be either a _Container_ or a _BasicItem_. The Item class represents an entity the user can interact with. Arbitrary examples may be: *swords, spoons, pens, etc..*. This class is abstract because an Item can be (exclusivly) one of two types: *Container* or *BasicItem*. 
+
+<u>*Name*</u> is a string that gives the entity a name. Pretty self explanatory. It is needed so that the game can tell the user (in words), what the item represents.
+
+<u>*ID*</u> is a uniquie integer which will differentiate all objects. This is needed to tell apart two objects with the same *Name*. 
+
+*<u>getName()</u>* returns the name of the Item
+
+<u>*getID()*</u> returns the ID of the item
+
+Item has a composition relation to (exclusivley) *Area* or a *Player*. This is because an Item can not both exist outside of the *Player's backpack* and in the surrounding area or vice versa. Furthermore, an Item can not exist without an *Area* or *Player's backpack* to exist in. 
+
+<h5 id="BasicItem">BasicItem</h5>
+
+Singelton is a subtype of <a href="#Item">Item</a> that a player can interact with. It is the most basic type of item. It inherits properties from Item, such as the Name and ID and their consequent getter functions. 
+
+An Item can be interracted with, or used to interract. To elaborate, imagine the command ***Attack chair with sword***. Here, the action attack has 2 clauses: the dependent and independent. This means that BasicItems need a way to know what can be done to it (ie be attacked), and what it can be used to do (ie attack). In this case, the BasicItem chair would have the string *Attack*, in its *UsedTo*.
```

Comment:
Here a glimpse of the game mechanics is presented. Please provide a more detailed explanation of how the game mechanics should work in the description of the system (assignment 1)